docs/core/testing/unit-testing-mstest-sdk.md

```diff
@@ -0,0 +1,103 @@
+
+## `MSTest.Sdk` with the `MSTest runner`
+
+The `MSTest.Sdk` supports the [`MSTest runner`](./unit-testing-mstest-runner-intro.md) and simplify the configuration of extensions to make the experience easy and smooth.
+
+How to use the `MSTest.Sdk` with the `MSTest runner` in a project:
+
+  ```xml
+  <Project Sdk="MSTest.Sdk/3.3.0">
+    <PropertyGroup> 
+        <TargetFramework>net8.0</TargetFramework>
+        <EnableMSTestRunner>true</EnableMSTestRunner> 
+        <OutputType>Exe</OutputType> 
+    </PropertyGroup>
+    <!-- references to the code to test -->
+  </Project>
+  ```
+
+ The `MSTest runner` comes with a set of built-in [extensions](./unit-testing-mstest-runner-extensions.md) that you can enable using an MSBuild property with the pattern `Enable[NugetPackageNameWithoutDots]`.  
+
+ For instance to enable the `MS Code Coverage` shipped as `Microsoft.Testing.Extensions.CodeCoverage` NuGet package you can use the property `<EnableMicrosoftTestingExtensionsCodeCoverage>true</EnableMicrosoftTestingExtensionsCodeCoverage>`:  
+
+  ```xml
+  <Project Sdk="MSTest.Sdk/3.3.0">
+    <PropertyGroup> 
+        <TargetFramework>net8.0</TargetFramework>
+        <EnableMSTestRunner>true</EnableMSTestRunner> 
+        <OutputType>Exe</OutputType> 
+    </PropertyGroup>
+
+    <!-- Extensions -->
+    <PropertyGroup>
+        <EnableMicrosoftTestingExtensionsCodeCoverage>true</EnableMicrosoftTestingExtensionsCodeCoverage>
+    </PropertyGroup>
+
+    <!-- references to the code to test -->
+  </Project>
+  ```
+
+### Available extensions
+
+| Extension package | MSBuild property |
+| -------- | ----------- |
+| `Microsoft.Testing.Extensions.CodeCoverage` | `<EnableMicrosoftTestingExtensionsCodeCoverage>true</EnableMicrosoftTestingExtensionsCodeCoverage>` |
+| `Microsoft.Testing.Extensions.TrxReport` | `<EnableMicrosoftTestingExtensionsTrxReport>true</EnableMicrosoftTestingExtensionsTrxReport>` |
+| `Microsoft.Testing.Extensions.CrashDump` | `<EnableMicrosoftTestingExtensionsCrashDump>true</EnableMicrosoftTestingExtensionsCrashDump>` |
+| `Microsoft.Testing.Extensions.HangDump` | `<EnableMicrosoftTestingExtensionsHangDump>true</EnableMicrosoftTestingExtensionsHangDump>` |
+| `Microsoft.Testing.Extensions.Retry` | `<EnableMicrosoftTestingExtensionsRetry>true</EnableMicrosoftTestingExtensionsRetry>` |
+| `Microsoft.Testing.Extensions.HotReload` | `<EnableMicrosoftTestingExtensionsHotReload>true</EnableMicrosoftTestingExtensionsHotReload>` |
+
+You can enable all the available extensions in "bulk" using the `<EnableAllTestingExtensions>true</EnableAllTestingExtensions>`
+
+```xml
+<Project Sdk="MSTest.Sdk/3.3.0">
+    <PropertyGroup> 
+        <TargetFramework>net8.0</TargetFramework>
+        <EnableMSTestRunner>true</EnableMSTestRunner> 
+        <OutputType>Exe</OutputType> 
+        <EnableAllTestingExtensions>true</EnableAllTestingExtensions>
+    </PropertyGroup>
+    
+    <!-- references to the code to test -->
+</Project>
+```
```

Comment:
Might be worth mentioning how you can change the extension version.

Reply:
I was wondering to keep it "hidden" for now and add when we will have a "more use cases" if we're able to keep this file "clean" users will bump all and get always correct/updated extensions, so pin a version should be "discouraged" if not in case of bugs and the need of quick fix/usage. Most probably we will ship all MSTest if the bug it "huge blocking" also if it's from an extension, what you think?